apps/web/src/data/subgraph/requests/daoQuery.ts

```diff
@@ -11,11 +11,6 @@ export type MyDaosResponse = Array<{
   chainId: CHAIN_ID
 }>
 
-const DAOS_TO_EXCLUDE = [
```

Comment:
Just out of curiousity, what are these and why did we remove the filter for them?

Reply:
@neokry mentioned that there were some bad deployments. Said to remove it as it was no longer needed.